react/src/pages/SessionLauncherPage.tsx

```diff
@@ -1216,7 +1221,7 @@ const SessionLauncherPage = () => {
                             <Flex direction="row">
                               {form.getFieldValue('environments')?.manual ? (
                                 <Typography.Text copyable code>
```

Comment:
I also found that there's a cosmetic issue when showing manual image in the last step of Neo session launcher. I already fixed it by wrapping with bracket. See the difference in the screenshot below:

### Screenshot(s)
| After | Before |
|------|--------|
| <img width="696" alt="Screenshot 2024-04-16 at 12 42 23 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/6607568d-75ef-41e4-89f1-0cc6c0737773"> | <img width="696" alt="Screenshot 2024-04-16 at 12 42 34 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/2723b0d8-9fd0-449e-b5f9-c98f24826790"> |

Reply:
🦅👀👍